client/src/pages/pot-instance/components/StatsContainer/StatsContainer.tsx

```diff
@@ -7,6 +7,7 @@ import CompletedStats from './CompletedStats/CompletedStats';
 import styles from './styles.module.css';
 
 const FOOD_EATEN = 'Food Eaten';
+const COUNT_OF_FOOD_EATEN = 'Count of Food Eaten';
```

Comment:
I was thinking more along the lines of `export const COUNT_OF_FOOD_EATEN = 'Count of Food Eaten';` so that it can be imported by CompletedStats component.